src/util/OAuth2Util.js

```diff
@@ -95,6 +95,43 @@ define(['okta', './Enums', './Errors'], function (Okta, Enums, Errors) {
     }
   };
 
+  // Parse through the OAuth 'authParams' object to ensure the 'openid' scope is
+  // included (if required)
+  util.scrubScopes = function (authParams) {
+    if (!authParams.responseType) {
+      return;
+    }
+
+    //  Convert scope into an Array
+    var scope = Array.isArray(authParams.scopes) ? authParams.scopes : authParams.scopes.split(' ');
```

Comment:
Chatted with Nate about this previously - his plan is to start moving devs to treat `scope` as a String as opposed to a String Array. As we build out new Sign-In helpers, all new functionality will be treated as such.

The rational behind it was developers will **only** be using *one* technique to render the Widget, so they *shouldn't* be confused.

Reply:
That makes sense, but I could still pass in scopes as an array and it would work.  So if we want to move to strings we should make sure that isn't possible?

Reply:
Because we're falling back to the `authParams` specified during Widget construction (passed as an `Array`, we need to account for both.

The `authParams` object via our new method `showSignInToGetTokens` will have `scope` returned as a string, while the traditional method will have `scopes` (**s**) as an Array.